provider/ansible.py

```diff
@@ -132,17 +132,17 @@ def python_install():
             LOG_JOB.error("Failed to get available pip binary")
             return False
         install_cmd = '%s install ansible' % pip_bin
+        # Install ansible depended packages that can't be installed
+        # automatically by pip when installing ansible
+        package_list = params.get('package_list', 'sshpass')
```

Comment:
The default pkg,  `sshpass` is not equal to `sshpass_pkg`.   With the design before, we use `sshpass_pkg` to specify the sshpass pkg to be installed,  it can be rpm link.  It was designed for the situation that no available pkg in distro. And pkg in distro with high priority.

Reply:
@yanan-fu understood, thanks for the explanation. But with current code, it would be simpler to be compatible with other packages; also in our test, we can replace the package name with a rpm link in 'package_list' in our internal file subtests-internal.cfg. @PaulYuuu What's your opinion?

Reply:
Yes